devfiles/java-mysql/devfile.yaml

```diff
@@ -24,7 +25,9 @@ components:
           -Duser.home=/home/user"
       - name: MAVEN_OPTS
         value: $(JAVA_OPTS)
-    memoryLimit: 700Mi
+      - name: PS1
+        value: $(echo ${0})\\$
```

Comment:
`PS1` should not be included, it is set in the entrypoint on the patched image.

Reply:
[ref](https://github.com/eclipse/che-devfile-registry/blob/92624598ba6cb07007bae1f0281262fa93e5a34b/arbitrary-users-patch/entrypoint.sh#L10)

Reply:
will remove :)